packages/@aws-cdk/aws-stepfunctions-tasks/test/lambda/integ.run-lambda.ts

```diff
@@ -0,0 +1,62 @@
+import { Code, Function, Runtime } from '@aws-cdk/aws-lambda';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as tasks from '../../lib';
+
+const app = new cdk.App();
```

Comment:
Nice integration test!

I would add a small comment like this - https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests - at the top. 

I believe the verification steps are the state machine should be in a completed state and not a failed state.

Reply:
added